tutorials/tutorial of qa for fast.ipynb

```diff
@@ -0,0 +1,313 @@
+{
```

Comment:
Never use spaces in any file names under any circumstances ever. Spaces are evil cancer that cause bugs